server/rest/dataset.py

```diff
@@ -21,7 +21,10 @@
 import shutil
 import pymongo
 import tempfile
-import json
+import json as Json
```

Comment:
This is a little confusing in the actual code, since `json` is the standard module name.  It would be better to rename variables rather than change the module (or import it as `json_module` to be more explicit about why it was done).